modules/web/cypress/e2e/v2/stories/ssh-keys.spec.ts

```diff
@@ -59,8 +59,16 @@ describe('SSH Key Management Story', () => {
     Pages.expect(View.Clusters.Default);
   });
 
-  it('should create the cluster with ssh key', {retries: 3}, () => {
+  it('should create the cluster with ssh key', () => {
+    Pages.Clusters.List.visit();
     Pages.Wizard.visit();
+
+    cy.get('.no-data-warning').then($warning => {
+      if ($warning.length > 0) {
+        cy.reload();
+      }
+    });
```

Comment:
should we add this to `settings/defaults.spec.ts` ?

Reply:
@ahmadhamzh did you mean `.no-data-warning` if class is removed or changed to something else it shouldn't fail or did you mean to keep trying multiple times and fail then?